2.6.md

```diff
@@ -850,6 +850,16 @@ This object contains any legal, governmental, or industry regulations that the s
     <td>string</td>
     <td>Communicates signals regarding consumer privacy under US privacy regulation. See <a href="https://github.com/InteractiveAdvertisingBureau/USPrivacy/blob/master/CCPA/US%20Privacy%20String.md">US Privacy String specifications</a>. Refer to Section 7.5 for more information.</td>
   </tr>
+  <tr>
+    <td><code>gpp</code></td>
+    <td>string</td>
+    <td>Contains the Global Privacy Platform’s consent string. See the <a href="https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform">Global Privacy Platform specification</a> for more details.</td>
+  </tr>
+  <tr>
+    <td><code>gpp_sid</code></td>
+    <td>integer array</td>
+    <td>Array of the section(s) of the string which should be applied for this transaction. Generally will contain one and only one value, but there are edge cases where more than one may apply. GPP Section 3 (Header) and 4 (Signal Integrity) do not need to be included. See the <a href="https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/main/Sections/Section%20Information.md">GPP Section Information</a> for more details.</td>
+  </tr>
```

Comment:
@drpaulfarrow

Would applicable sections and the string be sufficient in your view? I saw you guys were asking for additional info beyond the string.

In general the applicable section to me feels quite incomplete. At the time of a bid request each section may have status does apply, does not apply, or may apply but we don't yet know in examples such as the bid request occurred before the cmp determined the scope. I am not sure a list of applicable sections is useful without another flag indicating if the string is complete yet or not.